src/mock/main.py

```diff
@@ -0,0 +1,29 @@
+"""This module is for thread start."""
```

Comment:
This file should have a different name. For example, `kivy_main.py`. Also the docstring should say something like `Mock kivy app with mock threads`.